generators/server/templates/src/main/resources/config/_application.yml

```diff
@@ -123,7 +123,7 @@ jhipster:
         corePoolSize: 2
         maxPoolSize: 50
         queueCapacity: 10000
-    #cors: #By default CORS are not enabled. Uncomment to enable.
+    #cors: #By default CORS is disabled. Uncomment to enable.
```

Comment:
Argh! I didn't know we were in fact using `org.springframework.web.cors.CorsConfiguration`, so we can't modify it and add the `enable` property...